laos_climate_change/data_loaders.py

```diff
@@ -0,0 +1,27 @@
+# Verify world shapefile
+from os.path import exists
+from urllib.request import urlretrieve
+
+path_to_file = "C:/Users/camil/Dropbox/Servicios/OECD-GGGI/laos-climate-change/laos_climate_change/wb_countries_admin0_10m.zip"
+url = "https://datacatalogfiles.worldbank.org/ddh-published/0038272/DR0046659/wb_countries_admin0_10m.zip?versionId=2024-05-14T14:58:01.5696428Z"
+filename = "wb_countries_admin0_10m.zip"
+
+
+def download_shape_files():
```

Comment:
Re-use this function twice by allowing the user to pass two arguments:

1. Output directory, with default as `../data/shapefilies/data/`
2. While file to download

So the signature is:
`def download_shapefile(which, output_path='../data/shapefiles/data/`):`

Then add a check on which, like:
```
if which == 'laos':
    url = LAOS_URL
elif which == 'world':
    url = WORLD_URL
else:
    raise NotImplementedError()
```

Reply:
Understood! I addressed this and the previous comment in the latest commit.